packages/rancher-kiali-server/templates/crd-template/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 1.32.1
+version: 1.35.1
```

Comment:
Is this 1.35.1 instead of 1.35.0 because of the whole 1==100 situation ? Meaning if upstream is 0 we use 0-99.

Reply:
Correct, for the leading zero problem. We did this for the previous version of Kiali as well.